I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/story/introduction/prologue.rpy", line 134, in script
    amelie "It's been a while since I used %{amelie_profile.user_name}s. I used to love that character..."
ValueError: unsupported format character '{' (0x7b) at index 32

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/story/introduction/prologue.rpy", line 134, in script
    amelie "It's been a while since I used %{amelie_profile.user_name}s. I used to love that character..."
  File "C:\Users\carol\Downloads\renpy-8.1.3-sdk\renpy\ast.py", line 930, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "C:\Users\carol\Downloads\renpy-8.1.3-sdk\renpy\exports.py", line 1463, in say
    what = what % tag_quoting_dict
ValueError: unsupported format character '{' (0x7b) at index 32

Windows-10-10.0.22621 AMD64
Ren'Py 8.1.3.23091805
Production 1.0
Tue Feb 20 16:31:36 2024
